generator.yaml

```diff
@@ -28,8 +28,14 @@ resources:
           input_fields:
             PolicyName: Name
     hooks:
+      sdk_read_one_post_set_output:
+        template_path: hooks/policy/sdk_read_one_post_set_output.go.tpl
       sdk_create_post_set_output:
         template_path: hooks/policy/sdk_create_post_set_output.go.tpl
+      sdk_update_post_set_output:
+        template_path: hooks/policy/sdk_update_post_set_output.go.tpl
+    update_operation:
+      custom_method_name: customUpdatePolicy
```

Comment:
Please add a quick comment above the `custom_method_name: customUpdatePolicy` line that explains you're using a custom update method because there is no `UpdatePolicy` API call..